go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.0
 
 require (
 	aead.dev/minisign v0.2.1
-	github.com/bufbuild/buf v1.31.0
+	github.com/bufbuild/buf v1.31.1-0.20240508184409-79f9211ad6f0
```

Comment:
pulling in https://github.com/bufbuild/buf/pull/2956 which hasn't been released yet.